docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.0/master.adoc
+
+
+:_newdoc-version: 2.18.2
+:_template-generated: 2024-07-01
+
+:_mod-docs-content-type: REFERENCE
+
+[id="new-features-7-1-0_{context}"]
+= New features
+
+This section provides the new features and improvements of the {ProductFullName} 7.1.0.
+
+.A support for analyzing applications managed with Gradle has been added to {ProductShortName} 
```

Comment:
"A" is not needed I think .

"Support for analyzing applications managed with Gradle has been added to {ProductShortName}"

Reply:
This is also a developer Preview feature.

Reply:
Oki, will add the dev preview note, thank you!